doc/attributes.md

```diff
@@ -67,3 +67,47 @@ message:
 type checking makes it possible for the Istio system to statically detect or prevent many configuration errors.
+
+# Standard Istio Attribute Vocabulary
+
+In a standard Istio deployment (using the Istio proxy), the system will produce the following set of attributes. This is considered the canonical attribute set for Istio.
+
+| Name | Type | Description | Kubernetes Example |
+|------|------|-------------|--------------------|
+| source.ip | ip_address | The IP address of the client that sent the request. | 10.0.0.117 |
+| source.port | int64 | The port on the client that sent the request. | 9284 |
+| source.name | string | The fully qualified name of the application that sent the request. | redis-master.namespace.deps.cluster.local |
+| source.uid | string | A unique identifier for the particular instance of the application that sent the request. | redis-master-2353460263-1ecey.namespace.pods.cluster.local |
+| source.namespace | string | The namespace of the sender of the request. | namespace.cluster.local |
+| source.labels | map | A map of key-value pairs attached to the source. | |
+| source.user | string | The user running the target application. | service-account@namespace.cluster.local |
+| target.ip | ip_address | The IP address the request was sent to. | 10.0.0.104 |
+| target.port | int64 | The port the request was sent to. | 8080 |
+| target.service | dns_name | The hostname that was the target of the request. | my-svc.my-namespace.svc.cluster.local |
+| target.name | string | The fully qualified name of the application that was the target of the request. | |
+| target.uid | string | A unique identifier for the particular instance of the application that received the request. | |
+| target.namespace | string | The namespace of the receiver of the request. | |
+| target.labels | map | A map of key-value pairs attached to the target. | |
+| target.user | string | The user running the target application. | service-account@namespace.cluster.local |
+| origin.ip | ip_address | The IP address of the originating application | |
+| origin.name | string | The fully qualified name of the originating application | |
+origin.uid | string | A unique identifier for the particular instance of the application that originated the request. This must be unique within the cluster. | for in cluster | pod-id gives sufficient to ascertain all other attributes needed by the policy. |
+| origin.namespace | string | The namespace of the originator of the request. | |
+| origin.labels | map | A map of key-value pairs attached to the origin. | |
+| origin.user | string | The user running the originating application. | |
+| request.headers | map | A map of HTTP headers attached to the request. | |
+request.id | string | A unique ID for the request, which can be propagated to downstream systems. This should be a guid or a psuedo-guid with a low probability of collision in a | temporal window measured in days or weeks. | |
```

Comment:
i think we have a spurious `|` in between "collision in a" and "temporal window".